jib-maven-plugin/src/test/resources/projects/complex/pom.xml

```diff
@@ -0,0 +1,75 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.test</groupId>
+  <artifactId>hello-world</artifactId>
+  <version>1</version>
+
+  <properties>
+    <java.target.version>1.8</java.target.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <jib-maven-plugin.version>@@PluginVersion@@</jib-maven-plugin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.test</groupId>
+      <artifactId>dependency</artifactId>
+      <version>1.0.0</version>
+      <scope>system</scope>
+      <systemPath>${project.basedir}/libs/dependency-1.0.0.jar</systemPath>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>${jib-maven-plugin.version}</version>
+        <configuration>
+          <from>
+            <image>localhost:5000/distroless/java</image>
+            <auth>
+              <username>testuser</username>
+              <password>testpassword</password>
+            </auth>
+          </from>
+          <to>
+            <image>localhost:5000/complex-image</image>
+            <auth>
+              <username>testuser</username>
+              <password>testpassword</password>
+            </auth>
+          </to>
+          <container>
+            <useCurrentTimestamp>true</useCurrentTimestamp>
+            <args>An argument.</args>
+            <mainClass>com.test.HelloWorld</mainClass>
+            <jvmFlags>
+              <jvmFlag>-Xms512m</jvmFlag>
+              <jvmFlag>-Xdebug</jvmFlag>
+            </jvmFlags>
+            <ports>
+              <port>1000/tcp</port>
```

Comment:
The test should probably test for these ports actually being exposed.

Reply:
Doesn't that already happen here? https://github.com/GoogleContainerTools/jib/pull/814/files#diff-dd10efd511098bc1a27585924f70f8f1R247